src/index.js

```diff
@@ -144,6 +137,6 @@ function emptyStringWhenNull (value) {
 
 function encryptOperationValue (codec, operation) {
   return codec.encrypt(operation.value).then(function (cipher) {
-    return { ...operation, value: cipher }
+    return Object.assign(operation, { value: cipher })
```

Comment:
Good catch avoiding mutation above.... this one left out on purpose? 🤔 

```suggestion
    return Object.assign({}, operation, { value: cipher })
```